packages/shared/selectors/common.ts

```diff
@@ -0,0 +1,9 @@
+import { K8sResourceCommon } from "@openshift-console/dynamic-plugin-sdk";
+import * as _ from 'lodash';
+
+export const getNamespace = <A extends K8sResourceCommon = K8sResourceCommon>(value: A) =>
+  _.get(value, 'metadata.namespace') as K8sResourceCommon['metadata']['namespace'];
+
+
```

Comment:
```suggestion
```

Reply:
Is suggested changes is that there should not be any new line at the EOF?